File: m2/Personaje.py
class Personaje:
    def __init__(self, nombre, defensa, poder, salud):
        self.nombre = nombre
        self.defensa = defensa
        self.poder = poder
        self.salud = salud

    def mostrar_informacion(self):
        print(f"Nombre: {self.nombre}")
        print(f"Defensa: {self.defensa}")
        print(f"Poder: {self.poder}")
        print(f"Salud: {self.salud}")

# Ejemplo de uso
heroe = Personaje("Arthas", 50, 70, 100)
heroe.mostrar_informacion()

File: m2/Personaje.js
class Personaje {
    constructor(nombre, defensa, poder, salud) {
      this.nombre = nombre;
      this.defensa = defensa;
      this.poder = poder;
      this.salud = salud;
    }
  
    mostrarInformacion() {
      console.log(`Nombre: ${this.nombre}`);
      console.log(`Defensa: ${this.defensa}`);
      console.log(`Poder: ${this.poder}`);
      console.log(`Salud: ${this.salud}`);
    }
  }
  
  // Ejemplo de uso:
  const superman = new Personaje("Superman", 50, 70, 100);
  superman.mostrarInformacion();
  
  const ironMan = new Personaje("IronMan", 80, 90, 100);
  ironMan.mostrarInformacion();
  